Makefile

```
@@ -72,5 +72,7 @@ stats_evm_circuit: # Print a table with EVM Circuit stats by ExecState/opcode
 stats_copy_circuit: # Print a table with Copy Circuit stats by ExecState/opcode
 	@cargo test -p zkevm-circuits --features=test,warn-unimplemented get_copy_states_stats -- --nocapture --ignored
 
+evm_exec_steps_occupancy: # Print a table for each EVM-CellManager CellType with the top 10 occupancy ExecutionSteps associated
+	@cargo test -p zkevm-circuits --release get_exec_step_occupancy --features test -- --nocapture --ignored`
```

Typos break the command.
```suggestion
	@cargo test -p zkevm-circuits --release get_exec_steps_occupancy --features test -- --nocapture --ignored
```